.github/ISSUE_TEMPLATE/not_working.md

```diff
@@ -0,0 +1,44 @@
+---
+name: "\U00002757 Something is not working"
```

Comment:
this seems like a duplicate of a bug report, with the intention that someone isn't as sure about it? I'd probably aim to remove it in favour of just having one way to report bugs

Reply:
sorted out here 4ecfdb1